juniper_official/interface/collect-logical-interface-stats.rule

```diff
@@ -92,30 +96,42 @@
             }
             field interface-name {
                 sensor interfaces-oc {
-                    where "/interfaces/interface/@name =~ /{{input-ifd-name}}/";
+                    where "/interfaces/interface/@name =~ /{{inp-interface-name}}/";
                     path "/interfaces/interface/@name";
                 }
                 type string;
                 description "Interfaces to be monitored";
             }
             field sub-interface-index {
                 sensor interfaces-oc {
-                    where "/interfaces/interface/subinterfaces/subinterface/@index =~ /{{input-ifl-no}}/";
+                    where "/interfaces/interface/subinterfaces/subinterface/@index =~ /{{inp-sub-interface-index}}/";
                     path "/interfaces/interface/subinterfaces/subinterface/@index";
                 }
                 type string;
                 description "Interfaces to be monitored";
+            }             
+            field device-vendor {
+                constant {
+                    value "{{inp-device-vendor}}";
+                }
+                type string;                
+                description "Indicates device vendor";
             }
-            variable input-ifd-name {
+            variable inp-interface-name {
```

Comment:
Changing of variable names will cause upgrade failures if the rules is already instantiated on older release